src/lock_file/update.rs

```diff
@@ -1838,7 +1838,7 @@ async fn spawn_solve_conda_environment_task(
         .await
         .with_context(|| {
             format!(
-                "failed to solve '{}' for {}",
+                "failed to solve requirements of environment '{}' for platform '{}'",
```

Comment:
Adding 'requirements' here to more closely match the text raised by `resolve_pypi` below.